recipe/gen_patch_json.py

```diff
@@ -289,6 +311,7 @@ def _gen_patch_instructions(index, new_index, subdir):
         "remove": [],
     }
 
+    _add_broken_to_removals()
```

Comment:
This should only be done for `subdir`

Reply:
This call is in the diff function and so it only happens per subdir.

Reply:
See the function implementation itself.

Reply:
That's redundant downloading and computations.

Reply:
Sure I can fix that.